docs/TablePartitionsStorageGroups.md

```diff
@@ -0,0 +1,421 @@
+
+Example steps to configure initial database archive import when TPSG is enabled are outlined below.
+
+```sh
+helm install database stable/database \
+  --set database.te.replicas=0 \
+  --set database.sm.hotCopy.replicas=0 \
+  --set database.sm.noHotCopy.enablePod=false \
+  ...
+```
+
+```sh
+helm install sgeast stable/database \
+    --set database.sm.storageGroup.enabled=true \
+    --set database.primaryRelease=false \
+    --set database.sm.hotCopy.replicas=1 \
+    --set database.sm.noHotCopy.replicas=1 \
+    --set database.te.enablePod=false \
+    --set database.autoImport.type=backupset \
+    --set database.autoImport.source=<URL to backup of SGEAST storage group> \
+    -f values.yaml
+```
+
+```sh
+helm install sgwest stable/database \
+  --set database.sm.storageGroup.enabled=true \
+  --set database.primaryRelease=false \
+  --set database.sm.hotCopy.replicas=1 \
+  --set database.sm.noHotCopy.replicas=1 \
+  --set database.te.enablePod=false \
+  --set database.autoImport.type=backupset \
+  --set database.autoImport.source=<URL to backup of SGWEST storage group> \
+  -f values.yaml
+```
+
+Wait for all database storage groups to become _Available_ and scale up the TEs.
+
+```sh
+kubectl scale deployment -l domain=nuodb,database=demo,component=te --replicas 3
+```
+
+## Troubleshooting
+
+### Storage Group Assignment Mismatch
+
+The SM startup logs below show a storage group assignment mismatch.
+The archive is servicing storage group `SGEAST`, however, the SM has been reconfigured to service storage group `sgnyc`.
+
+```text
+2023-09-19T05:56:24.019+0000 INFO  [admin-nuodb-cluster0-0:sm-sgeast-nuodb-cluster0-demo-database-0] STDOUT Warning: Specified '--storage-groups' do not match storage groups for an existing archive
+    The following storage groups are served by the archive but were not supplied via '--storage-groups':
+        SGEAST
+    Use the 'remove storage-group' command if you wish for the archive to stop serving them
+    The following storage groups were supplied via '--storage-groups' but are not served by the archive
+        sgnyc
+    Use the 'add storage-group' command if you wish for the archive to start serving them
+```
+
+> **Action**: Storage group reconfiguration is unsupported. Revert the changes to the storage group name for this Helm release and install another Helm release for the new storage group. For more information, see [Storage Group Name Reconfiguration](#storage-group-name-reconfiguration).
+
+### Backup Failure
+
+The output below shows a failed _full_ backup:
+
+```text
+Executing full hotcopy as a prerequisite for incremental hotcopy: full hotcopy in backupset 20230919T053410 has failed
+Starting full backup for database demo on processes with labels 'role hotcopy' ...
+'backup database' failed: Failure while performing hot-copy: Not all storage groups are included in hot-copy for dbName=demo, missingSgs=[SGEAST]
+Error running hotcopy 1
```

Comment:
This "Error running hotcopy 1" line is coming from `kubectl exec`, right? I think you can drop it since it is not clear that you're executing this using `kubectl exec`.

Reply:
It is from `nuobackup` output. Actually, I can leave just the line indicating the error and strip the rest to avoid confusion.